docs/en/concepts-and-designs/mal.md

```diff
@@ -177,23 +177,19 @@ MAL should instruct meter-system how to do downsampling for metrics. It doesn't
 Down sampling operations are as global function in MAL:
 
  - avg
- - latest (TODO)
+ - sum
+ - latest
  - min (TODO)
  - max (TODO)
  - mean (TODO)
- - sum (TODO)
  - count (TODO)
 
 The default one is `avg` if not specific an operation.
 
-If user want get latest time from `last_server_state_sync_time_in_seconds`:
+If user want to get the latest time from `last_server_state_sync_time_in_seconds`:
 
 ```
-latest(last_server_state_sync_time_in_seconds.tagEqual('production', 'catalog'))
-
-or
-
-latest last_server_state_sync_time_in_seconds.tagEqual('production', 'catalog')
+last_server_state_sync_time_in_seconds.tagEqual('production', 'catalog').latest()
```

Comment:
This may be a small breaking change, the form of `latest xxxx` is now NOT supported, instead, one should use `xxx.latest()`. The form of `latest xxx` is error-prone b/c something like

```
latest (foo - 1).tagEqual('bar', '1').sum(['tt']).irate().histogram().histogram_percentile([50,99]).service(['rr'])
```

should be considered the `latest` value of `(foo - 1).tagEqual('bar', '1').sum(['tt']).irate().histogram().histogram_percentile([50,99]).service(['rr'])`, but it's actually the `histogram_percentile` of `latest((foo - 1).tagEqual('bar', '1')).sum(['tt']).irate().histogram()`.

Reply:
This looks good to me, @hanahmily any idea?

Reply:
LGTM. and you have to search our mal relevant rules to adapt to this change.

Reply:
Done, only unit tests used this